Geometry_Engine/Compute/BooleanDifference.cs

```diff
@@ -182,7 +143,7 @@ public static List<Polyline> BooleanDifference(this Polyline region, List<Polyli
             if (allRegions.Count == 1)
                 return new List<Polyline> { region };
 
-            Vector normal = region.Normal();
+            Vector normal = region.INormal();
```

Comment:
Why not `Normal`?

Reply:
Same answer to all of them, I trusted VS which complained for whatever reason while batch making changes, something which I thought I reverted in https://github.com/BHoM/BHoM_Engine/pull/1709/commits/f13df3464d3075f8ab5a5ed6716c3fdc92e24e14 but apparently not.

I'll undo these as well

Reply:
wait, most these were already removed, think you missed my last commit, and there's an other one coming. just going to double check through it all again first